src/stories/components/HorizontalTitle/HorizontalTitle.js

```diff
@@ -0,0 +1,45 @@
+import './styles.scss';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+/**
+ * HorizontalTitle Component
+ *
+ * @component
+ * @param {eyebrow} eyebrow
+ * @param {title} title
+ * @param {content} paragraph content
+ * @return {object}
+ */
+export const HorizontalTitle = (
+    { modifierClasses, eyebrow, title, content }) => {
+    return (
+        <div
+            className={
+                ['horizontal-title', `${modifierClasses}`].join(' ').trim()
+            }
+        >
+            {eyebrow ?
```

Comment:
To get the horizontal layout, you'll probably want to put a wrapper `div` around the title and `eyebrow` and set the display of the component to grid or flex for large tablets and above. Example:

```
.horizontal-title {
        @include bp-tablet-lg {
            display: grid;
            grid-gap: $spacing-3;
        }

        @include bp-desktop {
            grid-gap: $spacing-5;
        }
    }
```